api_scrape.py

```diff
@@ -1,17 +1,31 @@
-from bs4 import BeautifulSoup
-import urllib.request
-
+import requests
+from requests.exceptions import HTTPError
 
 urls = open('appIDs.txt', 'r')
 
-with urls as f:
-    url = f.readline()
-    req = urllib.request.Request(url)
-    resp = urllib.request.urlopen(req)
-    respData = resp.read()
-    print(respData)
 
+try:
+    with urls as f:
+        url = f.readline()
+        response = requests.get(url)
+        response.raise_for_status()
```

Comment:
I didn't know this was a feature of the module, nice find and very useful!

Reply:
Thanks! It's really really helpful and keeps things clean!